### ISSUE TITLE
wasi-libc upstreaming?

### ISSUE THREAD

**Issue by yamt** [dicej/component-linking-demo#4]
i think it makes sense to have a minimum version in wasi-libc as an experimental target.
how do you think?

**Comment by yamt** [comment]
> a minimum version 

i meant something like this:
https://github.com/yamt/wasi-libc/tree/so

**Comment by dicej** [comment]
Yes, I'm planning to upstream the changes to all the submodules in this repo.  Currently, I'm focusing on `wasm-tools`, but will work on `wasi-sdk` and `wasi-libc` after that,

**Comment by yamt** [comment]
> Yes, I'm planning to upstream the changes to all the submodules in this repo. Currently, I'm focusing on `wasm-tools`, but will work on `wasi-sdk` and `wasi-libc` after that,

ok.
if/when you want me to help or do wasi-libc part, please let me know.

**Comment by dicej** [comment]
FYI: https://github.com/WebAssembly/wasi-libc/pull/429 and https://github.com/WebAssembly/wasi-sdk/pull/338

**Comment by yamt** [comment]
> FYI: [WebAssembly/wasi-libc#429](https://github.com/WebAssembly/wasi-libc/pull/429) and [WebAssembly/wasi-sdk#338](https://github.com/WebAssembly/wasi-sdk/pull/338)

Thank you!
